package.json

```diff
@@ -37,7 +41,6 @@
     "semantic-release": "^24.0.0",
     "typescript": "^5.5.4",
     "vite": "^5.3.4",
-    "vite-plugin-dts": "^3.9.1",
```

Comment:
does this come as a transitive dependency from package-vite-config?

Reply:
Yes, it should be automatically brought in with the config package. 

https://github.com/lokalise/shared-ts-libs/blob/bc78b5021db08319a63bf0eb8c82b647614d669f/packages/dev/package-vite-config/package.json#L34